pack/core/core.json

```
@@ -424,6 +424,7 @@
         "position": 19,
         "quantity": 1,
         "skill_willpower": 1,
+        "tags": "hh.hd.",
```

Later you switch to the alphabetical 'hd.hh.' so I think change here too: 

```suggestion
        "tags": "hd.hh.",
```